src/ert/shared/storage/extraction.py

```diff
@@ -16,13 +16,13 @@ def _prepare_x_axis(x_axis: List[Union[int, float, str, pd.Timestamp]]) -> List[
     return [str(x) for x in x_axis]
 
 
-def _get_obs_data(key, obs) -> dict:
-    return dict(
-        name=key,
-        x_axis=obs.columns.get_level_values(0).to_list(),
-        values=obs.loc["OBS"].to_list(),
-        errors=obs.loc["STD"].to_list(),
-    )
+def _get_obs_data(key, obs) -> Mapping[str, Any]:
```

Comment:
Do you know if it would be possible and correct ` Mapping[Optional[str]]`

Reply:
`Mapping` should have two arguments, https://docs.python.org/3/library/typing.html#typing.Mapping , so `Optional[str]` would probably give an error. `str, Any` is not the same as `Optional[str]`, `[str, Any] `just means that the `Mapping` (the dict) is indexed by strings, but we don't specify the types these string keys link to.

Reply:
Interesting. Thanks for explaining!